entity-framework/core/modeling/spatial.md

```diff
@@ -108,16 +108,15 @@ Here is an example of using ProjNet4GeoAPI to calculate the distance between two
 ``` csharp
 static class GeometryExtensions
 {
-    static readonly IGeometryServices _geometryServices = NtsGeometryServices.Instance;
-    static readonly ICoordinateSystemServices _coordinateSystemServices
+    static readonly CoordinateSystemServices _coordinateSystemServices
         = new CoordinateSystemServices(
             new CoordinateSystemFactory(),
             new CoordinateTransformationFactory(),
             new Dictionary<int, string>
             {
                 // Coordinate systems:
 
-                // (3857 and 4326 included automatically)
+                [4326] = GeographicCoordinateSystem.WGS84.WKT,
```

Comment:
@airbreather Is it by design that these aren't included by default anymore? Or a bug?

Reply:
It was a consequence of [NetTopologySuite/ProjNet4GeoAPI@1f3b73d0](https://github.com/NetTopologySuite/ProjNet4GeoAPI/commit/1f3b73d0#diff-ea95da2fa6e030fd9ee31551f59f3c6f).  I feel like it might have been unintentional, but I can't say for sure, since EPSG isn't the only authority...

@FObermaier thoughts?  Specifically, we've changed the behavior when someone passes in a non-`null` `IEnumerable<KeyValuePair<int, string>>` to one of these constructors.  Previously, we would have auto-included SRIDs 3857 and 4326 without needing the user to give us the WKT for them, but now it's up to the user to add these themselves if they want them.

Reply:
(while I was looking at that diff, I noticed a race condition in the old code that this change would have solved, which leads me to believe that the old behavior was actually what was unintentional)